Code-4-Community/constellation
# Add Routes

### ℹ️ Issue

Closes #22 

### 📝 Description

- Added routes for the two pages (grant form page under `/form` and view forms for authenticated user under `/all-forms`)
- Added a catch all route that displays a simple 404 page

### ✔️ Verification

- Run `nx serve frontend`
- Navigate to `localhost:4200/form` to view the form page
- Navigate to `localhost:4200/all-forms` and verify if I'm signed out, it redirects to the Login page first

<img width="1299" alt="image" src="https://user-images.githubusercontent.com/55663526/230405990-a3598842-1c73-4f6d-9023-8af368c33824.png">

<img width="579" alt="image" src="https://user-images.githubusercontent.com/55663526/230412783-7bc5b281-bf0d-433f-b55b-d89c82745c7c.png">


### 🏕️ (Optional) Future Work / Notes

Prettier 404 page?

Not sure how our app will integrate with NEGC but our 404 might not be necessary if we're sharing a domain name and they're gonna implement 404 and a catch all route.
